app/main.py

```diff
@@ -1,3 +1,69 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> int():
```

Comment:
Wrong return type annotations it returns Distance
Also, write int() is so bad....

Reply:
yep I write Distance but PyCharm give me ERROR